docs/swagger.yaml

```diff
@@ -1244,7 +1244,64 @@ paths:
         5XX:
```

Comment:
If this file just contains some descriptions. I have no opinions.
Otherwise, just omit my review.